lib/themes/login_theme.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+class LoginTheme extends Theme {
```

Comment:
The theme should not be only Login specific. Rename it in MainTheme, or LightAppTheme.

Contributors module is supposed to be a package later, that's why i named a specific Theme for it.